build/git.go

```diff
@@ -94,7 +95,7 @@ func getGitAttributes(ctx context.Context, contextPath string, dockerfilePath st
 	}
 
 	if setGitLabels {
-		if root, err := gitc.RootDir(); err != nil {
+		if root, err := gitc.WorkTree(ctx); err != nil {
```

Comment:
I think we still need the `RootDir` func otherwise `root` will always be empty here.